swap_meet/item.py

```diff
@@ -1,2 +1,36 @@
+
+
 class Item:
-    pass
\ No newline at end of file
+    
+    def __init__(self, category = "", condition = 0, age = None):
+        self.category = category
+        self.condition = condition
+        self.age = age
+
+    #override str() for Item, returns "Hello World!" instead
+    def __str__(self):
+        return "Hello World!"
+
+
+    #describes condition based on the value, range from 0 to 5
+    def condition_description(self):
+
+        if self.condition == 0:
+            return "Terrible condition"
+        
+        if self.condition == 1:
+            return "Bad condition"
+
+        if self.condition == 2:
+            return "Slightly bad condition"
+
+        if self.condition == 3:
+            return "Acceptable condition"
+
+        if self.condition == 4:
+            return "Good condition"
+
+        if self.condition == 5:
+            return "Mint conditon"
```

Comment:
The condition can be a float, so it can contain a decimal.  In the cases where the condition has a decimal, this function will return None.  There are a few solutions, one is to cast the condition to an int, which will drop the decimal portion of the number and round down.   Another possible solution is to test for a range of values (ex: `if self.condition <= 5 and self.condition > 4`.